example/e2e/asyncstorage.spec.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+describe('Async Storage', () => {
+  let restartButton;
+  let closeKeyboard;
+  let test_getSetClear;
+  let test_mergeItem;
+
+  beforeAll(async () => {
+    await device.reloadReactNative();
+    restartButton = await element(by.id('restart_button'));
```

Comment:
Consider adding a test that would fail if restartButton didn’t actually restart. I believe all of the tests in this file would continue passing if restartButton was a no-op.